package.json

```diff
@@ -6,12 +6,13 @@
   "devDependencies": {
     "@types/koa": "^2.0.51",
     "@types/node": "^12.12.6",
+    "nodemon": "^1.19.4",
     "ts-node": "^8.4.1",
     "typescript": "^3.7.2"
   },
   "scripts": {
     "build": "tsc",
     "start": "node ./build/index.js",
-    "start:dev": "node --require ts-node/register ./src/index.ts"
+    "start:dev": "nodemon --exec 'node --require ts-node/register ./src/index.ts' --ext .js,.json,.ts"
```

Comment:
I remember having problems before when trying `nodemon` instead a Docker container, but seems ok at the moment?